oam/etc/Columnstore.xml

```diff
@@ -137,6 +137,7 @@
 		<BulkRollbackDir>/var/lib/columnstore/data1/systemFiles/bulkRollback</BulkRollbackDir>
 		<MaxFileSystemDiskUsagePct>98</MaxFileSystemDiskUsagePct>
 		<CompressedPaddingBlocks>1</CompressedPaddingBlocks> <!-- Number of blocks used to pad compressed chunks -->
+        <FastDelete>false</FastDelete>
```

Comment:
JFYI Some time ago there was an issue updating in deb-based distros. Basically dpkg thinks the Columnstore.xml was manually changed(even if it didn't) thus adding new options into the config forces automatic(that is w/o human intervention)dpkg upgrades to fail waiting for a confirmation. We can take the risk and see if it fail or not.

Reply:
I thought we had resolved that issue

Reply:
I see. I was not aware of this issue of package install failing if we add a new option to Columnstore.xml. I can remove this option from the .xml. In which case, the customer will have to add this option back in on all nodes in the cluster and restart, if they want to enable `FastDelete`. By default, if this option is not present in the .xml file, it will be treated as false.

Reply:
Let it be here and see what the server's CI tells us.